src/clientSide.ts

```diff
@@ -0,0 +1,83 @@
+import { zencode_exec } from "zenroom";
+import { UserChallenges } from "./model/userChallenges";
+import {
+  fileExists,
+  readJSONFromFile,
+  readStringFromFile,
+} from "./service/fileService";
+import * as dotenv from "dotenv";
+
+const DEFAULT_CLIENT_SIDE_CONTRACT =
+  "./node_modules/keypair-lib/lib/zencode/Keypair-Creation-Client-Side.zen";
```

Comment:
Feel that all this hardcoded paths, will lead to bad things ;p I get that are just as a default fallbacks... 
But does make sense to you to have at least a common, let's say BASEPATH, like 
`DEFAULT_CONTRATCTS_BASEPATH="./node_modules/keypair-lib/lib/zencode/"`
So in case of any refactoring, there is a single source of truth/change?

Reply:
We need to use a basepath that has https://nodejs.org/api/modules.html#modules_dirname instead of node_modules

Reply:
Changed to const DEFAULT_CLIENT_SIDE_CONTRACT =__dirname+"/zencode/Keypair-Creation-Client-Side.zen";